codecarbon/emissions_tracker.py

```diff
@@ -203,13 +212,13 @@ def _measure_power(self) -> None:
         self._last_measured_time = time.time()
 
 
+@suppress(Exception)
```

Comment:
This decorator doesn't work at the class level. It should be left in the constructor method
```suggestion
```